toolkit/tools/scheduler/schedulerutils/printresults.go

```diff
@@ -372,6 +393,16 @@ func printErrorInfoByCondition(condition bool, format string, arg ...any) {
 	}
 }
 
+// printWarningInfoByCondition prints warning or info level logs depending on the input condition.
+// If the condition is true, it prints a warning level log and an info level one otherwise.
+func printWarningInfoByCondition(condition bool, format string, arg ...any) {
```

Comment:
Is this function being used anywhere? I couldn't find a usage

Reply:
https://github.com/microsoft/azurelinux/pull/9885

Reply:
Remove it from this PR then?